src/GraphQLMiddleware.php

```diff
@@ -67,12 +61,7 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
     }
 
     private function isGraphQLRequest(ServerRequestInterface $request) {
-        return $this->hasUri($request) || $this->hasGraphQLHeader($request);
```

Comment:
But in your README example, you pipe at the specific URI `/graphql`.

You wrote in your first comment :
> If this is not for us, we should quit as soon as possible. 

Could you give an example where we should not handle a request sent to `/graphql` ?